nimble/controller/src/ble_ll_conn_hci.c

```diff
@@ -76,7 +75,7 @@ ble_ll_init_alloc_conn_comp_ev(void)
     rc = 0;
     evbuf = g_ble_ll_conn_comp_ev;
     if (evbuf == NULL) {
-        evbuf = ble_hci_trans_buf_alloc(BLE_HCI_TRANS_BUF_EVT_HI);
+        evbuf =ble_transport_alloc_evt(0);
```

Comment:
I liked how original defines for hi/low made code easy to read,   maybe add BLE_HCI_TRANS_EVT_DISCARDABLE or similar?

Reply:
the parameter name is `discardable` so it's pretty clear what `0` and `1` mean here